supervision/dataset/formats/yolo.py

```diff
@@ -153,7 +153,7 @@ def load_yolo_annotations(
     image_paths = [
         str(path)
         for path in list_files_with_extensions(
-            directory=images_directory_path, extensions=["jpg", "jpeg", "png"]
+            directory=images_directory_path, extensions=["*"]
```

Comment:
There is a small but important side-effect of this change. If there are other files than images in the directory, we will also try to load them. For example, macOS puts a `.DS_Store` file in the directory. @patel-zeel I suggest putting here a list of image extensions that you have tested.

Reply:
I agree. On it!